tests/app/service/test_rest.py

```diff
@@ -133,16 +132,8 @@ def test_get_service_by_id(admin_request, sample_service):
     assert json_resp['data']['dvla_organisation'] == '001'
     assert json_resp['data']['sms_sender'] == current_app.config['FROM_NUMBER']
     assert json_resp['data']['prefix_sms'] is True
-
-
-def test_get_service_by_id_returns_free_sms_limit(admin_request, sample_service):
-    json_resp = admin_request.get('service.get_service_by_id', service_id=sample_service.id)
-    assert json_resp['data']['free_sms_fragment_limit'] == current_app.config['FREE_SMS_TIER_FRAGMENT_COUNT']
-
-
-def test_get_detailed_service_by_id_returns_free_sms_limit(admin_request, sample_service):
-    json_resp = admin_request.get('service.get_service_by_id', service_id=sample_service.id, detailed=True)
-    assert json_resp['data']['free_sms_fragment_limit'] == current_app.config['FREE_SMS_TIER_FRAGMENT_COUNT']
```

Comment:
We need this. Updating free_sms_fragment)limit still go through the service_api_client in admin.